sourmash/sbtmh.py

```diff
@@ -107,6 +107,33 @@ def _max_jaccard_underneath_internal_node(node, hashes):
     return max_score
 
 
+def _similarity_downsample(mh_1, mh_2):
```

Comment:
This look like something that could be merged into the `similarity` method for minhashes?